src/scrabble.js

```diff
@@ -1,5 +1,150 @@
-function scrabble() {
-  // write code here
+function checkIncorrect(letterArray) {
+  const allowedChars = [
```

Comment:
better put this outside function since its const and read only:

`SCABBLE_LETTERS` or similar would be better name